### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -475,6 +502,16 @@ You can subscribe on the following `$SYS` topics to get client presence:
...
 
+## Plugins
+
+- [aedes-persistence](https://github.com/moscajs/aedes-persistence): In-memory implementation of an Aedes persistence
+  - [aedes-persistence-mongodb](https://github.com/moscajs/aedes-persistence-mongodb): MongoDB persistence for Aedes
+  - [aedes-persistence-redis](https://github.com/moscajs/aedes-persistence-redis): Redis persistence for Aedes
+  - [aedes-persistence-level](https://github.com/moscajs/aedes-persistence-level): LevelDB persistence for Aedes
+  - [aedes-persistence-nedb](https://github.com/ovhemert/aedes-persistence-nedb#readme): NeDB persistence for Aedes
+- [aedes-logging](https://github.com/moscajs/aedes-logging): Logging module for Aedes, based on Pino
+- [aedes-stats](https://github.com/moscajs/aedes-stats): Stats for Aedes
```
Comment (mcollina) [inline README.md]: We should probably include links to the mqemitter equivalents as well.

File: README.md [hunk 2]
```diff
@@ -475,6 +502,16 @@ You can subscribe on the following `$SYS` topics to get client presence:
  - `$SYS/+/disconnect/clients` - will inform about client disconnections.
 The payload will contain the `clientId` of the connected/disconnected client
 
+## Plugins
```
Comment (mcollina) [inline README.md]:
```suggestion
## Persistences
```

Comment (robertsLando) [reply README.md]: @mcollina I think plugins is better as stats and logs are not persistences

File: README.md (hunk 1 above)
Comment (robertsLando) [reply README.md]: Now I will add missings mqemitters links. COuld you check also if all the ticks on the table of features are correct? I have a '?' on bridge as I don't understand what it means

Comment (robertsLando) [reply README.md]: done

**Comment by robertsLando** [comment]
Just for info. I will always submit PRs and I will let you review and merge them once you decide they are good

File: README.md (hunk 1 above)
Comment (qwertangel) [reply README.md]: readme has '?' on bridge. what does it mean ?

**Comment by robertsLando** [comment]
https://github.com/mqtt/mqtt.github.io/wiki/bridge_protocol

File: README.md (hunk 1 above)
Comment (qwertangel) [reply README.md]: what i mean to highlight is - aedes either supports bridge or not. so documentation should represent that accurately.

**Comment by robertsLando** [comment]
Actually nope

Daniel

**Comment by qwertangel** [comment]
so a 'X' instead of '?' would be more appropriate in documentation ?

**Comment by robertsLando** [comment]
Yes, initially when I made this PR I didn’t know if it was or not that is why I placed that question mark

**Pull request merged**